README.md

```diff
@@ -23,6 +23,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 2. Short array syntax (`[]` instead of `array()`) MUST be used.
 3. All PHP code MUST adhere to the [PSR-4 Autoloading](http://www.php-fig.org/psr/psr-4/) Standard.
 4. Variable names MUST be camelCase and not snake_case or StUdLyCaSe or UpperCamelCase.
+  1. Acronyms MUST be treated as normal words. `xmlHttpRequest` not `XMLHTTPRequest`.
```

Comment:
PSR-1 actually: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#1-overview

Neither PSR-1 or PSR-2 specific a standard for variable naming, which is why it's explicitly defined in ours.